### PR TITLE
Fixing path not being found

### PR DESCRIPTION
**Pull request by lock9**
Fixing error found in #129.
Before, usign ```dotnet src\Neo.Compiler.MSIL\bin\Debug\netcoreapp2.0\neon.dll  .\TestContract\bin\Debug\netstandard2.1\TestContract.dll``` would cause an error.

Before:
```
PS C:\Users\HP\Workspace\neo-devpack-dotnet> dotnet src\Neo.Compiler.MSIL\bin\Debug\netcoreapp2.0\neon.dll  .\TestContract\bin\Debug\netstandard2.1\TestContract.dll
Neo.Compiler.MSIL console app v2.4.1.0
Open File Error:System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\HP\Workspace\neo-devpack-dotnet\TestContract\bin\Debug\netstandard2.1\TestContract\bin\Debug\netstandard2.1\TestContract.dll'.
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at System.IO.File.OpenRead(String path)
   at Neo.Compiler.Program.Main(String[] args) in C:\Users\HP\Workspace\neo-devpack-dotnet\src\Neo.Compiler.MSIL\Program.cs:line 58
```

After:
```
PS C:\Users\HP\Workspace\neo-devpack-dotnet> dotnet C:\Users\HP\Workspace\neo-devpack-dotnet\src\Neo.Compiler.MSIL\bin\Debug\netcoreapp2.0\neon.dll C:\Users\HP\Workspace\neo-devpack-dotnet\TestContract\bin\Debug\netstandard2.1\TestContract.dll
Neo.Compiler.MSIL console app v2.4.1.0
Find entrypoint:System.Object Neo.Compiler.MSIL.TestClasses.Contract1::Main(System.String,System.Object[])
convert succ
gen abi succ
write:TestContract.nef
write:TestContract.abi.json
write:TestContract.manifest.json
SUCC
``` 

EDIT: I used the wrong command when reporting, but the problem exists

### CODE REVIEW

File: src/Neo.Compiler.MSIL/Program.cs [hunk 1]
```diff
@@ -55,7 +57,7 @@ public static void Main(string[] args)
             //open file
             try
             {
-                fs = File.OpenRead(filename);
+                fs = File.OpenRead(dllName);
```
Comment (shargon) [inline src/Neo.Compiler.MSIL/Program.cs]: Why this change if filename should be good?

**Comment by lightszero** [comment]
TestContract had be removed.

**Comment by lock9** [comment]
Hi @lightszero, I created another one. The problem exists, the path of the dll and pdb are not matching depending from where you run neon

**Comment by lock9** [comment]
@shargon  I will close this PR, can you apply the fix on #131 ? Thanks
